drsm/chunk.go

```diff
@@ -95,6 +98,7 @@ func (c *chunk) ReleaseIntID(id int32) {
 	}
 }
 
+// chunkid-123456
 func getChunIdFromDocId(id string) int32 {
 	logger.DrsmLog.Infof("id received: %v value", id)
```

Comment:
Does this have to be `info` level? I am asking this because I see the current `amf` log has 11K+ of these message logs (e.g., `id received: chunkid-9097`)

Reply:
Updated. Though I am unclear why would that happen. Need to debug it further.